s64da_benchmark_toolkit/netdata.py

```diff
@@ -73,11 +73,12 @@ def get_system_stats(self, df, resolution):
     def _write_stats_per_query(self, df, output):
         data = self._get_netdata_per_query(df, output)
         query_ids = natsorted(data.keys())
+        flname, flext = os.path.splitext(output)
```

Comment:
It would make more sense to move this logic of name change to the place where you call this method. 
As a user of the method, I would expect the output I give it to be respected and not modified, therefore it would be better to modify it before calling the method.

Reply:
done